src/App.vue

```diff
@@ -1,10 +1,23 @@
 <template>
-  <div id="nav">
-    <router-link to="/">Home</router-link> |
-    <router-link to="/about">About</router-link>
-  </div>
-  <router-view/>
-</template>
+  <Layout>
```

Comment:
Can file name be lower case here and other places?

Reply:
https://v3.vuejs.org/style-guide/#component-files-strongly-recommended Naming for file is PascalCase or always kebab-case. Updated to kebab-case.